src/components/ec-alert/ec-alert.story.js

```diff
@@ -0,0 +1,104 @@
+import { storiesOf } from '@storybook/vue';
+import { boolean, text } from '@storybook/addon-knobs';
+import EcAlert from './ec-alert.vue';
+
+const stories = storiesOf('Alert', module);
+
+stories.add('basic', () => ({
+  components: { EcAlert },
+  props: {
+    title: {
+      default: text('title', 'Error!'),
+    },
+    subtitle: {
+      default: text('subtitle', 'Something was wrong with the update.'),
+    },
+    type: {
+      default: text('type', 'error'),
+    },
+    textButton: {
+      default: text('textButton', 'Read more'),
+    },
+    dismissable: {
+      default: boolean('dismissable', true),
+    },
+  },
+  data() {
+    return { show: true };
+  },
+  template: `
+    <div>
+    <ec-alert :title="title" :subtitle="subtitle" :type="type" :dismissable="dismissable" :button-text="textButton" v-model="show">
+    </ec-alert>
+
+    <button @click="show = !show">{{ show ? 'Hide' : 'Show' }}</button>
+    </div>`,
+}));
+
+stories.add('all', () => ({
+  components: { EcAlert },
+  data() {
+    return {
+      showInfo: true,
+      showSuccess: true,
+      showWarning: true,
+      showError: true,
+      showInfo1: true,
+      showSuccess1: true,
+      showWarning1: true,
+      showError1: true,
+    };
+  },
+  template: `
+  <div>
+      <ec-alert title="Info Alert" type="info" />
```

Comment:
honest question, could we use a Set with all the configs, and iterate? Or do we prefer to be verbose and explicit writing every combination?

Reply:
I don't have strong opinion about that, but maybe we can set the config for that and don't write one by one combination for all the components.

But honestly I was following the current version for that in the button component, but as I said, I'm happy to change it! :smile:

Reply:
What do you think guys @mcibique @stavros-tomas ?

Reply:
Buttons we should rename the "all story" as it become much more complicated as initially intended and we don't cover every possible combination in the "all story". 

It should be a per component case scenario. If the developer feels will be faster to iterate via a config/loop lets do like that, don't copy paste from other stories.

Reply:
We should rather generate it, imagine adding new type into this.

Reply:
Done! If you guys can take a look :P